tox.ini

```diff
@@ -30,10 +21,12 @@ setenv =
     clocale: LC_CTYPE = C.ascii
     clocale: LC_ALL = C
     devdeps: PIP_EXTRA_INDEX_URL = https://pypi.anaconda.org/scipy-wheels-nightly/simple
+    PYTHONIOENCODING = utf-8
+
 
 # Run the tests in a temporary directory to make sure that we don't import
 # astropy from the source tree
-changedir = .tmp/{envname}
+change_dir = .tempdir
```

Comment:
I would rather have https://github.com/tox-dev/tox/issues/2620 resolved before we merge this and commit the empty dir to the git tree.

Reply:
sure, but there are more issues. So to discover those, I'll leave it in until fixed.

Reply:
I just tested tox-dev/tox#2643, and that fixes the directory creation (with `change_dir = .tmp/{envname}`)

Reply:
4.0.3 is out with this fix